src/collective/cover/static/css/layout_edit.css

```diff
@@ -731,3 +731,45 @@ input[type='submit'].btn::-moz-focus-inner
 {
     cursor: move;
 }
+body .cssclasswidget
```

Comment:
how are we going to deal with intermediate caching servers and this change?

Reply:
the simplest solution in this case is to rename the resource name appending the modify date.